src/actions/houseHoldActions.js

```diff
@@ -10,7 +10,7 @@ export const INVITE_MEMBER = 'INVITE_MEMBER';
 const fetchHousehold = () => (dispatch) => {
   dispatch({ type: LOADING });
   return axiosWithAuth()
-    .get('/members/household/assignable')
+    .get('/members/household')
```

Comment:
Either the backend will need to be updated before we merge this or we can remove this change temporarily

Reply:
It can wait then. I only added it because you asked me to.

Reply:
I'm going to go ahead and approve, but we'll hold off on merging at the moment